### PR TITLE
Merge back hotfix to develop

### PR DESCRIPTION
**Pull request by haammar-ledger**
### 📝 Description

Conflicts prevented `hotfix` to be merged into `develop`, needed to solve them manually.

### ❓ Context

- **Impacted projects**: all 
- **Linked resource(s)**: n/a 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/cli/CHANGELOG.md [hunk 1]
```diff
@@ -30,13 +44,16 @@
   - @ledgerhq/live-common@27.4.0-next.0
   - @ledgerhq/cryptoassets@6.36.0-next.0
 
+<<<<<<< HEAD
+=======
```
Comment (lambertkevin) [inline apps/cli/CHANGELOG.md:48]:
```suggestion
```
Missed conflicts 👀

**Review by lambertkevin — APPROVED** [review]
Just 2 missed conflicts

File: apps/ledger-live-desktop/src/renderer/components/FirebaseFeatureFlags.tsx [hunk 2]
```diff
@@ -23,6 +23,21 @@ const checkFeatureFlagVersion = (feature: Feature) => {
...
+  ) {
+    return {
+      enabledOverriddenForCurrentDesktopVersion: true,
+      ...feature,
+      enabled: false,
+    };
+  }
+  return feature;
+};
+
```
Comment (thomasrogerlux) [inline apps/ledger-live-desktop/src/renderer/components/FirebaseFeatureFlags.tsx:40]: Duplicate, the same function is just above

**Review by thomasrogerlux — CHANGES_REQUESTED** [review]

File: apps/ledger-live-desktop/src/renderer/components/FirebaseFeatureFlags.tsx (hunk 2 above)
Comment (thomasrogerlux) [reply apps/ledger-live-desktop/src/renderer/components/FirebaseFeatureFlags.tsx:40]:
```suggestion
```

Comment (haammar-ledger) [reply apps/ledger-live-desktop/src/renderer/components/FirebaseFeatureFlags.tsx]: Thanks !

**Review by elbywan — APPROVED** [review]
lgtm 👍

**Review by hedi-edelbloute — APPROVED** [review]

**Pull request merged**